atlp-rwanda/falcons-ec-be
# #184637234 Configure The App to use PostgreSQL as DB Engine and Sequelize ORM

#### What does this PR do?

- Configures the project to use postgrSQL as the database and Sequelize ORM

#### Description of Task to be completed?
- Configure The App to use PostgreSQL as DB Engine and Sequelize ORM
#### How should this be manually tested?
 - clone the project and use the terminal to start the development server by "npm run dev"
 - make sure that you have PostgreSQL installed
 - create a database and
 - then use the database name and password to make the DEV_DATABASE_URL in your env file,
 - refer to the env.example file
#### Any background context you want to provide?
 none
#### What are the relevant pivotal tracker/Trello stories?
 Tracker: 184637234
#### Screenshots (if appropriate)
 none
#### Questions:
 none
